evm/deploy-its.js

```diff
@@ -168,14 +176,15 @@ async function deployAll(config, wallet, chain, options) {
                 );
             },
         },
+        // Deploy only the appropriate token implementation based on chain type
         interchainToken: {
             name: 'Interchain Token',
-            contractName: 'InterchainToken',
+            contractName: interchainTokenContractName,
```

Comment:
```suggestion
            contractName: 'InterchainToken',
```
should still keep the name the same, so the config doesn't use chain specific names. Otherwise, all scripts reading it need to be specialized